src/Core/Mesh/TriangleMesh.hpp

```diff
@@ -106,24 +114,34 @@ class TriangleMesh {
         return m_vertexAttribs.addAttrib<T>( name );
     }
 
-    /// Remove attribute by handle, invalidate the handles to this attribute.
+    /// Remove attribute by handle.
     /// \see AttribManager::removeAttrib() for more info.
     template <typename T>
     void removeAttrib( AttribHandle<T> h ) {
         m_vertexAttribs.removeAttrib( h );
     }
 
-    /// Copy only the required attributes (deep copy).
-    /// The position and normal attributes are always copied, no need to provide their handles.
+    /// Erases all attributes, leaving the mesh with the topology and geometry only.
+    inline void clearAttributes();
+
+    /// Copy only the mesh topology and geometry.
+    /// The needed attributes can be copied through copyAttributes().
+    inline void copyBaseGeometry( const TriangleMesh& other );
```

Comment:
There is a side effect in this function: all the attributes previously existing in the instance are destroyed.
This must be specified in the documentation.

Reply:
true